ui-metalk8s-client-generator/src/generateClient.ts

```diff
@@ -55,6 +55,22 @@ export function generateClient(crdFile: string, destinationFile: string) {
       return { error };
     }
   }
+
+  export async function get${singleName}(${singleName}Name: string): Promise<Result<${singleName}>> {
+    if (!customObjects) {
+      return { error: 'customObject has not yet been initialized' };
+    }
+    try {
+      return await customObjects.getClusterCustomObject(
+        '${crdSpec.spec.group}',
+        '${crdSpec.spec.version}',
+        '${crdSpec.spec.names.plural}',
+        ${singleName}Name,
+      );
+    } catch (error) {
+      return { error };
+    }
+  }
```

Comment:
Maybe this part should be squashed with `ui: add a flow typed client generator` (it's more than just `ui: use generated metalk8s client and types`)